packages/formation/sass/base/_b-variables.scss

```diff
@@ -113,72 +122,49 @@ $color-cool-blue-lightest:  #dce4ef;
 // Places where DSVA is going rogue!
 //===================================
 
+
 $color-black:                #000;
 /* 
 ADDED TO CSS-LIBRARY
 file: tokens/color.json
 issue: vets-design-system-documentation#2042
 */
+
 $color-link-default-hover:   rgba($color-black, 0.05);
 
-$color-green-darker:         #195c27;
-$color-va-accent:            #988530; // New gold "metal" acccent
-$color-gold:                 #fdb81e;
-$color-feedback-warning-background:      #fac922; // Used for Disability benefits
-$color-gray-light-alt:       #eee;
 $color-va-modal-bg:          rgba($color-gray-dark, .8);
 
-$color-inset-bg:             #e0f3f8;
-$color-gibill-accent:        #fff1d2; // TODO: Replace with $color-gold-lightest;
-
 $color-white-alpha70:        rgba(255,255,255,.7);
 
 $color-sidebar-nav-bg:   rgba($color-black, 0.25);
 
-$color-orange: #EB7F29;
-$color-gold-darker: #988530;
-
 //===================================
 // Hub Icon colors for BC
 //
 //===================================
 
-$red-70v: #8b1303;
-$red-60v: #b51d09;
-$indigo-cool-60: #3f57a6;
-$red-60: #a23737;
-$mint-cool-50: #21827f;
-$orange-warm-50: #bd5727;
-$green-cool-50: #3e8520;
-$gold-50: #8e704f;
-$green-warm-50: #6f7a41;
-$blue-50: #357ab2;
-$violet-warm-60: #864381;
-
-// Contextual variable names
-$color-hub-health-care: $indigo-cool-60;
-$color-hub-education: $mint-cool-50;
-$color-hub-disability: $red-60;
-$color-hub-careers: $orange-warm-50;
-$color-hub-pension: $green-cool-50;
-$color-hub-housing: $gold-50;
-$color-hub-life-insurance: $green-warm-50;
-$color-hub-burials: $blue-50;
-$color-hub-records: $violet-warm-60;
-$color-hub-family-member: $color-primary-darker;
-$color-hub-service-member: $color-primary-darker;
+$color-hub-health-care: #3f57a6;
+$color-hub-education: #21827f;
+$color-hub-disability: #a23737;
+$color-hub-careers: #bd5727;
+$color-hub-pension: #3e8520;
+$color-hub-housing: #8e704f;
+$color-hub-life-insurance: #6f7a41;
+$color-hub-burials: #357ab2;
```

Comment:
This should be #2378c3 (blue-50). I guess the blue-50 that was in here was incorrect for burials.